src/app/contribute/contribution-process/file-issue/index.html

```diff
@@ -24,4 +29,37 @@
     </li>
   </ol>
 
+  <stache-page-anchor>
+    Report bugs
+  </stache-page-anchor>
+
+  <p>
+    You can also file issues to report any bugs that you uncover while you work with a package. We use issues to track and assign work.
+  </p>
+<ol>
+    <li>
+      <p>
+        Create a GitHub issue in the <a stacheRouterLink="/contribute/packages">applicable repo</a>.
+      </p>
+      <sky-restricted-view>
+        <sky-alert alertType="info">
+          Blackbaud developers should report bugs in ADO instead of GitHub. The SKY UX team uses ADO dashboards to manage work, but other than the location for filing issues, the process remains the same, including the availability of a <a href="https://dev.azure.com/blackbaud/Products/_workitems/create/Bug?templateId=762f6a46-de2a-49ee-b2c4-c3bb0ef60de0&ownerId=26bb0bf6-45ac-482d-b33e-02e252c297e3">template for filing bugs</a>.
+        </sky-alert>
+      </sky-restricted-view>
+      <sky-restricted-content-alert></sky-restricted-content-alert>
```

Comment:
Not a huge deal - but wanted to ask. We currently have two alerts in two different places on this page. Is that what we want or do we just want to place one alert at the top of the page about content existing for BB employees?

Reply:
I removed the second alert on this topic.